src/app/pages/home/about/about.component.html

```diff
@@ -1 +1,41 @@
-<div>About Page</div>
\ No newline at end of file
+<div class="aboutus-overlay1">
+    <div class="aboutus-overlay1-content">
```

Comment:
I recommend internal `container-fluid` inside `containers` as they will respect the parent `container`'s restriction on the side margins. Use `container-fluid` instead for this class and center its content using the `text-center` class. We can add bottom spacing to split the _headline_ from the 4-way grid by using `mb-5`.

```suggestion
    <div class="container-fluid text-center mb-5">
```